### DISCUSSION TITLE
How can I disable social part of facebook?

### DISCUSSION THREAD

**Discussion by takeda** [community/community#56278] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Is Microsoft trying to turn Github into a Facebook?
Seriously?

I just want to see updates on the repos that I starred, the issues and PRs that I opened or subscribed to.

I don't give a damn that one person starred another. Why there's the tendency to turn everything into social media and ruin it?

**Comment by seuribe** [comment]
Same here. I don't want a feed first things I see when I enter GitHub. I want to see my repositories. I came here to work.
